### ISSUE TITLE
Runtime error: module compiled against API version a but this version of numpy is 9

### ISSUE THREAD

**Issue by machomachopadre** [BVLC/caffe#4104]
Hey everyone :wave: :wave: !! Earlier today I ran _import caffe_ in ipython. Then I got the ever popular _numpy.core.multiarray failed to import_. Sigh. I looked at all the countless threads on it and got ONE solution to work. Press enter, run it again and act as if it never happened(somehow it "solves" itself on its own). After importing caffe I ran the classic _python/classify.py --print_results example/images/cat.jpg foo_ (caffe veterans and beginners alike know what it is). Now the error comes and bites me back. The first line is _Runtime error: module compiled against API version a but this version of numpy is 9_ and the last line is-you guessed it!- _numpy.core.multiarray failed to import_ :angry: I believe this is caused by anaconda installing another numpy in its folder or when I installed scimage(with _pip install sci-something*_) it might have installed a conflicting numpy. Side note: I installed numpy using _apt-get install python-numpy_. HELP and THANKS.
tl;dr (I really needed one of these): The error is in the title and I think it is because of anaconda and/or scimage installing a conflicting version
*i tend to forget stuff in about 4-6 seconds

**Comment by seanbell** [comment]
Please ask for help with installation and libraries on the mailing list.  The problems you are having are with python libraries, not with caffe.

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.

**Comment by machomachopadre** [comment]
okie thanks

**Comment by machomachopadre** [comment]
where should i post this any suggestions?

**Comment by seanbell** [comment]
These aren't issues with caffe; they are with python and anaconda.  You could try asking any general programming forum.  You should start by looking at your python path, and also checking for the physical locations of the numpy libraries (you can do this in python after importing numpy).

**Comment by 123chengbo** [comment]
I have met this problem too，have you solve it？
I will very appreciate if you response to me，Thank you
